### PR TITLE
clean(er) query

### PR DESCRIPTION
**Pull request by SadCheeto**
## Describe your changes
changed query so it is easier to exchange
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

File: metrics/complex_logic.py [hunk 1]
```diff
@@ -19,12 +19,10 @@ def _parse(self, ast):
         count = 0
```
Comment (FrederikRothe) [inline metrics/complex_logic.py:19]: duplicate count = 0?

File: metrics/complex_logic.py [hunk 2]
```diff
@@ -19,12 +19,10 @@ def _parse(self, ast):
...
-                right: (boolean_operator) @function.boolean_operator)
-
-            """
+        (_	[
+            condition: (boolean_operator)
+            right: (boolean_operator)] @bool_operator)
+        """
         )
         captures = query.captures(ast.root_node)
         count = 0
```
Comment (FrederikRothe) [inline metrics/complex_logic.py:28]: dupe?

**Review by FrederikRothe — APPROVED** [review]
Good job, 2 comments.

File: byoqm/main.py [hunk 3]
```diff
@@ -46,6 +47,14 @@
...
     default=datetime(9999, 1, 1),
 )
+@click.option(
+    "--verbose",
+    "-v",
+    type=bool,
+    required=False,
+    help="Extra logging",
+    default=False,
+)
```
Comment (toffernator) [inline byoqm/main.py:57]: Is this branch based off of #52?

Comment (SadCheeto) [reply byoqm/main.py:57]: yes

**Pull request merged**